### DISCUSSION TITLE
Discord link issue

### DISCUSSION THREAD

**Discussion by reza-ts** [implerhq/impler.io#799] [Q&A] [answered]
Could you please check the discord link in README? I get a Cloudflare error following it and can't access.

**Comment by chavda-bhavik** [comment]
Yes @r-yeganeh Thanks for checking

**Comment by chavda-bhavik** [comment]
@r-yeganeh Working on it. Meanwhile, can you join Discord using the following link, https://discord.gg/JSKvqmXQW3?

I am excited to welcome you!

**Reply by chavda-bhavik** [reply, marked as answer]
Link started working!

**Reply by reza-ts** [reply]
Great!
